geode-membership/src/main/java/org/apache/geode/distributed/internal/membership/gms/membership/GMSJoinLeave.java

```diff
@@ -428,6 +432,24 @@ public boolean join() throws MemberStartupException {
     }
   }
 
+  boolean hasCoordinatorJoinedCluster(int viewId, long retrySleep)
```

Comment:
looks like this extraction was test driven

Reply:
How about naming this something indicating that a sleep is going to be performed, like "pauseIfThereIsNoCoordinator"?